src/components/MoviesList/MoviesList.scss

```diff
@@ -1 +1,5 @@
-// 
+.movies {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+  gap: 20px;
+}
```

Comment:
add following code in style block so that all cards looks even
.card-image img {
  object-fit: contain;
}